ns_server_static/fts/fts.js

```diff
@@ -43,6 +43,12 @@ import {newParsedDocs} from "./fts_easy_parse.js";
 import {newEditFields, newEditField} from "./fts_easy_field.js";
 import {newEasyMappings, newEasyMapping} from "./fts_easy_mapping.js";
 
+import ftsListTemplate from "./fts_list.html";
```

Comment:
any generic code refactoring could be a separate change? can u please explain the context here?